src/controllers/midi/hss1394controller.cpp

```diff
@@ -191,6 +189,7 @@ void Hss1394Controller::send(QByteArray data) {
     int bytesSent = m_pChannel->SendChannelBytes(
         (unsigned char*)data.constData(), data.size());
 
+    controllerDebug(formatSysexMessage(getName(), data));
     //if (bytesSent != length) {
```

Comment:
I don't remember. Am I listed as the last person to touch that line? If so, maybe the HSS1394 spec said that it wasn't implemented and always returns 0 or something.